File: fix-translations.js
#!/usr/bin/env node

const fs = require('fs');
const path = require('path');

// Path to the translations file
const filePath = path.join(__dirname, 'src', 'locales', 'translations.ts');

console.log('Reading file:', filePath);

// Read the file
let content = fs.readFileSync(filePath, 'utf8');

console.log('File size:', content.length, 'chars');

// Count occurrences before
const beforeCount = (content.match(/\\\\\\\\n\\\\\\\\n/g) || []).length;
console.log('Found', beforeCount, 'occurrences of \\\\\\\\n\\\\\\\\n');

// Replace \\\\n\\\\n with \\n\\n (4 backslashes to 2 backslashes)
// In the file, \\\\n\\\\n appears as literal text, so we need to replace it
content = content.replace(/\\\\\\\\n\\\\\\\\n/g, '\\\\n\\\\n');

// Count occurrences after  
const afterCount = (content.match(/\\\\\\\\n\\\\\\\\n/g) || []).length;
console.log('Remaining occurrences:', afterCount);

// Write the file back
fs.writeFileSync(filePath, content, 'utf8');

console.log('✅ File fixed successfully!');
console.log('Replaced', (beforeCount - afterCount), 'occurrences');

File: fix_translations.py
#!/usr/bin/env python3

import re

# Read the file
with open('src/locales/translations.ts', 'r', encoding='utf-8') as f:
    content = f.read()

print(f'File size: {len(content)} chars')

# Count occurrences before
before_count = len(re.findall(r'\\\\\\\\n\\\\\\\\n', content))
print(f'Found {before_count} occurrences of \\\\\\\\n\\\\\\\\n')

# Replace \\\\n\\\\n with \\n\\n
# In the file these appear as 4 backslashes followed by n, so we replace with 2
content = content.replace('\\\\\\\\n\\\\\\\\n', '\\\\n\\\\n')

# Count occurrences after
after_count = len(re.findall(r'\\\\\\\\n\\\\\\\\n', content))
print(f'Remaining occurrences: {after_count}')

# Write the file back
with open('src/locales/translations.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print('✅ File fixed successfully!')
print(f'Replaced {before_count - after_count} occurrences')
